### ISSUE TITLE
Getting unexpected value 'SocialLoginModule'

### ISSUE THREAD

**Issue by joncorrin** [abacritt/angular4-social-login#5]
Hi, 

I used you work around for the AOT compilations. 

Im receiving an error when trying to push to production. 

```
remote:        ERROR in Unexpected value 'SocialLoginModule in /tmp/build_3c514eb8278c6a857f76b5d21d5ae9f7/node_modules/angular4-social-login/dist/lib/sociallogin.module.d.ts' imported by the module 'AppModule in /tmp/build_3c514eb8278c6a857f76b5d21d5ae9f7/src/app/app.module.ts'. Please add a @NgModule annotation.
remote:        
remote:        ERROR in ./src/main.ts
remote:        Module not found: Error: Can't resolve './$$_gendir/app/app.module.ngfactory' in '/tmp/build_3c514eb8278c6a857f76b5d21d5ae9f7/src'
remote:        @ ./src/main.ts 3:0-74
remote:        @ multi ./src/main.ts

```

Was wondering what solutions I have? 

I dont see any issues as far as the code written. Seems to be pretty straigthforward on your docs. 

Here is my service.ts

```
import { BrowserModule } from '@angular/platform-browser';
import {NgModule} from '@angular/core';
import { FormsModule } from "@angular/forms"
import { AppComponent } from './app.component';
import {AppRoutingModule} from "./app-routing.module";
import {UserService} from "./user/user.service";
import {LandingPageComponent} from "./landing-page/landing-page.component";
import {HttpClientModule} from "@angular/common/http";
import {NavComponent} from "./nav/nav.component";
import {FooterComponent} from "./footer/footer.component";
import {UiService} from "./ui.service";
import {environment} from "../environments/environment";
import {ContactService} from "./contact/contact.service";
import {ContactComponent} from "./contact/contact.component";
import { SocialLoginModule, AuthServiceConfig, GoogleLoginProvider, FacebookLoginProvider } from "angular4-social-login";

let config = new AuthServiceConfig([
  {
    id: GoogleLoginProvider.PROVIDER_ID,
    provider: new GoogleLoginProvider(environment.providers.google.clientId)
  },
  {
    id: FacebookLoginProvider.PROVIDER_ID,
    provider: new FacebookLoginProvider(environment.providers.facebook.clientId)
  }
]);

export function provideConfig() {
  return config;
}

@NgModule({
  declarations: [
    AppComponent,
    LandingPageComponent,
    NavComponent,
    FooterComponent,
    ContactComponent
  ],
  imports: [
    BrowserModule,
    FormsModule,
    HttpClientModule,
    AppRoutingModule,
    SocialLoginModule

  ],
  providers: [
    UserService,
    UiService,
    ContactService,
    {
      provide: AuthServiceConfig,
      useFactory: provideConfig
    }
  ],
  bootstrap: [AppComponent]
})
export class AppModule {}


```

**Comment by jaibatrik** [comment]
#3 also talks about this. I'll try to see this over the weekend.

**Comment by jaibatrik** [comment]
Should get resolved with the latest commit (v1.0.12 in npm). Reopen if you still see the issue.
